test/index.test.js

```diff
@@ -64,12 +66,6 @@ describe('Outbound', function() {
     });
   });
 
-  describe('loading', function() {
-    it('should load', function(done) {
-      analytics.load(outbound, done);
-    });
-  });
```

Comment:
it was breaking the tests but after attempting to fix them, I realized that we were testing it for the sake of it being part of our code and not that we needed to use it.